src/components/index.ts

```diff
@@ -1,3 +1,4 @@
 export { Avatar, EditButton } from './Avatar';
 export { default as Modal } from './Common/Modal/Modal';
```

Comment:
첫 번째 줄 Avatar도 Modal과 같은 위치인
`./Common/Avatar` 에서 불러와야 오류가 안나는 것 같습니다!

Reply:
확인해보겠습니다!

Reply:
[dev에서는 Avatar가 common에 있지 않아서](https://github.com/1e5i-Shark/algobaro-fe/tree/dev/src/components) 오류가 나지 않고 있는데
이번 PR에서는 common 폴더로 옮겨주셔서
경로를 수정해주셔야 할 것 같아요!!

```suggestion
export { Avatar, EditButton } from './Common/Avatar';
export { default as Modal } from './Common/Modal/Modal';
```